### ISSUE TITLE
Bookmark Icon on Console Panel

### ISSUE THREAD

**Issue by sxandy** [anakinsk/SafariStand#15]
Bookmark Icon in toolbar on Console Panel not visible.

**Comment by Fofer** [comment]
Yes, same here.  There icon is invisible but clicking on the assigned space for it still works.

**Comment by Fofer** [comment]
http://i.imgur.com/Ik8uEJA.png

**Comment by anakinsk** [comment]
Is that still an issue?

**Comment by sxandy** [comment]
Yes it‘s not a problem in functionality it’s a UI thing.

Alex
